distributions/distribution-resources/src/main/resources/etc/org.ops4j.pax.logging.cfg

```diff
@@ -28,7 +28,7 @@ log4j.appender.out.maxBackupIndex=10
 # File appender - events.log
 log4j.appender.event=org.apache.log4j.RollingFileAppender
 log4j.appender.event.layout=org.apache.log4j.PatternLayout
-log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%-20.20c{1}] - %m%n
+log4j.appender.event.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss.SSS} [%c{1}] - %m%n
```

Comment:
> I would use the full qualified name

We are only talking about the events.log here, where the logger name is ALWAYS smarthome.events.<something>.